packages/core/src/store/actions.ts

```diff
@@ -2,10 +2,9 @@ import type {Coordinates, UniqueIdentifier} from '../types';
 import type {DroppableContainer} from './types';
 
 export enum Action {
-  DragStart = 'dragStart',
+  SetInitiailCoordinates = 'setInitialCoordinates',
```

Comment:
```suggestion
  SetInitialCoordinates = 'setInitialCoordinates',
```
small typo

Reply:
thanks, can't commit as is - need to rename all the usages.
I will do it later

Reply:
@xkill-9  done